### DISCUSSION TITLE
Licensing information inconsistent

### DISCUSSION THREAD

**Discussion by jeisemancdb** [SixLabors/ImageSharp#2196] [Ideas]
Apologies if this is the incorrect place to put this, the other options didn't sound correct either. Just wanted to bring to your attention that the information in which this library is licensed are inconsistent between your NuGet page and your website. NuGet readme implies that it is licensed under the normal Apache 2.0 license, however on your website it is a modified/split license.

**Comment by dlemstra** [comment]
This was already answered here: https://github.com/SixLabors/ImageSharp/issues/2188

**Reply by JimBobSquarePants** [reply]
@jeisemancdb Is really important that you read before you ask, especially before you misuse the features category in the forums.
